torch/onnx/symbolic_opset9.py

```diff
@@ -1157,6 +1157,19 @@ def _unique2(g, input, sorted, return_inverse, return_counts):
     globals()[name] = parse_args('v', 'i')(partial(sym_help._cast_func_template, v))
 
 
+@parse_args('v', 'i', 'v', 'v', 'v', 'v')
+def empty(g, sizes, dtype, layout, device, pin_memory=False, memory_format=None):
+    return g.op("ConstantOfShape", sizes,
```

Comment:
This looks exactly the same as zeros/zeros_like. Consider reusing the code?

Reply:
updated, thanks!